src/Dialog/ConfirmationDialog.tsx

```diff
@@ -1,5 +1,5 @@
 import React, {useCallback} from 'react'
-import ReactDOM from 'react-dom'
+import {createRoot} from 'react-dom/client'
```

Comment:
I'm not sure if this is SSR-compatible, given that it uses the `react-dom/client` API. Can anyone advise on that?

Reply:
Sounds like it's good to go then!